parser/parser.go

```diff
@@ -26,6 +34,244 @@ func (f File) String() string {
 	return sb.String()
 }
 
+func (f File) CreateRequest() (*api.CreateRequest, error) {
+	req := &api.CreateRequest{}
+
+	var messages []api.Message
+	var licenses []string
+	params := make(map[string]any)
+
+	home, err := os.UserHomeDir()
+	if err != nil {
+		return nil, err
+	}
+
+	for _, c := range f.Commands {
+		switch c.Name {
+		case "model":
+			path := c.Args
+			if path == "~" {
+				path = home
+			} else if strings.HasPrefix(path, "~/") {
+				path = filepath.Join(home, path[2:])
+			}
```

Comment:
nit: could we just expand `~` once vs doing this in both branches?

Reply:
I realized this was broken anyway since it would match `~someotheruser` so ended up refactoring it.